packages/sample-app/sample_app/strands_example.py

```
@@ -0,0 +1,410 @@
+
+Note: Itinerary customized based on {interests} preferences."""
+    
+    add_to_memory("planning_agent", "itinerary_created", {"destination": destination, "duration": duration, "itinerary": itinerary})
+    shared_memory["trip_plans"][destination] = itinerary
+    return itinerary
+
+
+@tool
+def estimate_budget(destination: str, duration: str, travelers: int = 1) -> str:
+    """Estimate travel budget for the trip."""
+    base_cost = 150  # per day per person
```

Ruff (0.12.2)

Local variable `base_cost` is assigned to but never used

Remove assignment to unused variable `base_cost`

(F841)